airflow/providers/amazon/aws/sensors/athena.py

```diff
@@ -50,19 +50,19 @@ class AthenaSensor(BaseSensorOperator):
     @apply_defaults
     def __init__(self,
                  query_execution_id,
-                 max_retires=None,
+                 max_retries=None,
```

Comment:
This is probably not a backward-compatible change. Please add a note in UPDATING.md

Reply:
Just added a note. Let me know if it needs more detail!